optaweb-employee-rostering-frontend/src/store/employee/employee.test.ts

```diff
@@ -276,19 +274,21 @@ describe('Employee reducers', () => {
     expect(
       reducer(state.employeeList, actions.addEmployee(addedEmployee)),
     ).toEqual({ ...state.employeeList,
-      employeeMapById: mapWithElement(storeState.employeeList.employeeMapById, addedEmployee) });
+      employeeMapById: storeState.employeeList.employeeMapById
+        .set(addedEmployee.id as number, mapDomainObjectToView(addedEmployee)) });
   });
   it('remove employee', () => {
     expect(
       reducer(state.employeeList, actions.removeEmployee(deletedEmployee)),
     ).toEqual({ ...state.employeeList,
-      employeeMapById: mapWithoutElement(storeState.employeeList.employeeMapById, deletedEmployee) });
+      employeeMapById: storeState.employeeList.employeeMapById.delete(deletedEmployee.id as number) });
```

Comment:
Alternatively, you can say `delete(deletedEmployee.id!)`. The exclamation mark is a non-null assertion. It's more concise and maybe a more precise type assertion. But we have an ESLint rule that complains about that. Dunno which of the two is worse.